pagopa/io-app
# feat: [PE-622] Change labels in CGN KO pages

## Short description
Change labels in CGN KO pages

## List of changes proposed in this pull request
- update translation files
- removed a button instance

## How to test

for all screens the steps are
- login into the app
- go to wallet section "Portafoglio"
- press add to Wallet "Aggiungi al portafoglio"
- press Carta Giovani Nazionale
- press activate/request card "Richiedi carta"
- check the copy on the outcome page

the codes that the backend should return for every page to be displayed listed below

- [figma non hai requisiti](https://www.figma.com/design/yOoBGDr1MJpa0LY5lDUFSd/IOapp_CartaGiovaniNazionale?node-id=6102-36447&t=BrVSSM5jZ6ZPyxMO-4) POST /api/v1/cgn/activation 403
- [figma fase attivazione](https://www.figma.com/design/yOoBGDr1MJpa0LY5lDUFSd/IOapp_CartaGiovaniNazionale?node-id=6102-36448&t=BrVSSM5jZ6ZPyxMO-4) POST /api/v1/cgn/activation 202
- [figma gia attiva](https://www.figma.com/design/yOoBGDr1MJpa0LY5lDUFSd/IOapp_CartaGiovaniNazionale?node-id=6102-36451&t=BrVSSM5jZ6ZPyxMO-4) POST /api/v1/cgn/activation 409
- [figma lavorando richiesta](https://www.figma.com/design/yOoBGDr1MJpa0LY5lDUFSd/IOapp_CartaGiovaniNazionale?node-id=6102-36449&t=BrVSSM5jZ6ZPyxMO-4) GET /api/v1/cgn/activation != 200
- [figma non disponibile](https://www.figma.com/design/yOoBGDr1MJpa0LY5lDUFSd/IOapp_CartaGiovaniNazionale?node-id=6102-36450&t=BrVSSM5jZ6ZPyxMO-4) POST /api/v1/cgn/activation 500
